### PR TITLE
Favourite button and Favourite Page

### LINKED ISSUE #19: Favourite button&Favourite Page
As a user, i want to select favorite and see in the favourite page

## Description
image

## Acceptance Criteria
- [ ] The icon will change when the user presses the button.
- [ ] When user click to favourite Button, user should be able to go to Favourite Page
- [ ] User can see the Products.

## Tasks
- [ ] Create feature branch
- [ ] Create Favourite Page Component
- [ ] Write functions
- [ ] Add button icons
- [ ] Update State
- [ ] Write test
- [ ] Add style

### PR DESCRIPTION
**Pull request by suademre**
When the user clicks on the favourite button on the products, the user can see the product they favourite on the like page.
 When the user likes the product, the favourite button changes.
Create feature branch
 Create Favourite Page Component
 Write functions
 Add button icons
 Update State
 Write test
 Add style

This resolved #19

### CODE REVIEW

File: src/components/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import IconButtons from 'components/IconButtons/IconButtons';
+import styled from 'styled-components/macro';
+
+function Card({product, favourites, handleFavoriteButtonClick}) {
+    return (
+        <CardDiv>
+            <CardImg src={product.imageFrond} alt={product.title} />
```
Comment (StefanBehring) [inline src/components/Card/Card.js:8]: still the typo `imageFront` ;)

File: src/components/IconButtons/IconButtons.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import {FaHeart} from 'react-icons/fa';
+import styled from 'styled-components/macro';
+
+function IconButtons({product, handleFavoriteButtonClick, active}) {
+    return (
+        <>
+            <FavoriteButton
+                active={active}
+                onClick={(e) => {
+                    handleFavoriteButtonClick(product);
```
Comment (StefanBehring) [inline src/components/IconButtons/IconButtons.js:11]: You dont need the parameter here. It's not wrong, but it's not really necessary
```
onClick={() => {
                    handleFavoriteButtonClick(product);
```

File: src/components/Card/Card.spec.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            title: 'Flying Low',
+            price: 105,
+            imageFrond:
+                'https://www.kaft.com/static/images/cache/800/tisort_flyinglow_19186_800_800.jpg?cacheID=1629358586000',
+            slug: 'flying-low',
+        },
+    ];
+
+    it('render card', () => {
+        const handleFavoriteButtonClick = jest.fn();
```
[the remark this answers is not in the document]
Comment (suademre) [reply src/components/Card/Card.spec.js:26]: thanx for your advice :)

File: src/components/IconButtons/IconButtons.js (hunk 2 above)
Comment (suademre) [reply src/components/IconButtons/IconButtons.js:11]: i agree with you :)  i forgot to delete

**Pull request merged**